tests/py/translations_test.py

```diff
@@ -0,0 +1,45 @@
+from qiskit_sphinx_theme.translations import get_language_label, get_translation_url
+
+
+def test_get_translation_url() -> None:
```

Comment:
this looks like a lot of assertions for similar things, I'm not experienced with pytest but you mentioned there is a way to use parameters in the test? Would there be a way to parameterise this test? Also could you add docstrings to the tests like we do in the other repos?